code/game/objects/structures/crates_lockers/closets/secure/secure_closets.dm

```diff
@@ -61,6 +61,7 @@
 		for(var/mob/O in viewers(user, 3))
 			if((O.client && !( O.blinded )))
 				to_chat(O, "<span class='notice'>The locker has been [locked ? null : "un"]locked by [user].</span>")
+				playsound(src.loc, 'sound/machines/click.ogg', 15, 1, -3)

```

Comment:
NO.

This means the sound will play, at the locker, for each of the people who can view it.

This would be like having `for(var/mob/M in view(user, 7) visible_message("you're a butt")`

that would mean if you had 50 mobs in view of the `user` it's going to spam the chat, 50 times, for everyone, with "you're a butt".